### PR TITLE
Add license headers

### PR DESCRIPTION
**Pull request by donce**

### CODE REVIEW

**Comment by Waldz** [comment]
Need Travis check for future generations. See ideas in MYST-487

File: client/connection/interface.go [hunk 1]
```diff
@@ -1,3 +1,20 @@
+/*
+ * Copyright (C) 2018 The Mysterium Network Authors
```
Comment (Waldz) [inline client/connection/interface.go]: Should text same as in `LICENCE` file

**Comment by tadovas** [comment]
Also sign-off check in github?

**Comment by donce** [comment]
@Waldz @tadovas you're talking about other subtasks of MYST-487. This is just one of subtasks, MYST-242.

**Comment by donce** [comment]
I'll add instructions with screenshots on how to setup GoLand for this after we've merged this PR :)

**Comment by tadovas** [comment]
@Waldz got it. LGTM

**Pull request merged**